src/main/flight/mixer.c

```diff
@@ -566,10 +580,95 @@ static void airplaneMixer(void)
 }
 #endif
 
+#ifdef USE_SERVOS
+uint8_t hasTiltingMotor(){
+	return currentMixerMode == MIXER_QUADX_TILT || currentMixerMode == MIXER_OCTOX_TILT;
+}
+
+/*
+ * return a float in range [-PI/2:+PI/2] witch represent the actual servo inclination wanted
+ */
+float getTiltServoAngle(void) {
+    float userInput = 0;
+
+    //get wanted position of the tilting servo
+    if (rcData[tiltArmConfig->channel] >= rxConfig->midrc) {
+    	userInput = rcData[tiltArmConfig->channel];
+    } else {
+    	userInput = rcData[PITCH];
+    }
+
+    //convert to radiant, keep eventual non-linearity of range
+    float servoAngle = scaleRangef(userInput, 1000, 2000, degreesToRadians(servoConf[TILTING_SERVO].minLimit), degreesToRadians(servoConf[TILTING_SERVO].maxLimit) );
+
```

Comment:
with reference to https://github.com/cleanflight/cleanflight/pull/928#discussion_r31756353

```
float servoAngle = scaleRangef(
    userInput, 
    1000, 
    2000, 
    degreesToRadians(0 - servoConf[TILTING_SERVO].minLimit),
    degreesToRadians(servoConf[TILTING_SERVO].maxLimit)
);
```